sigle/src/modules/publicStory/components/PublicStory.tsx

```diff
@@ -173,6 +174,10 @@ const CustomStyle = createGlobalStyle<{ siteColor?: string }>`
     `}
 `;
 
+const DynamicTipTapEditor = dynamic<{ story: Story; editable: boolean }>(() =>
```

Comment:
we dynamically load the editor so it won't be loaded on pages using the old slate editor content, that way we don't ship a big bundle size on all pages